### PR TITLE
[xwf-m] xwf-m gateway with unique server id 

### PR DESCRIPTION
**Pull request by ymasmoudi**
## Summary
Interrogates xwf-m orc8r and ensure a unique ID for each gateway.
Update hostname for the current host

## Test Plan
tested on xwfmqa1

### CODE REVIEW

**Comment by ymasmoudi** [comment]
@AyliD with snowflake fix

**Review by mpgermano — CHANGES_REQUESTED** [review]
I don't support putting orc8r admin rest certs on the gateways. This would allow for a lot more damage if any gateway were to be compromised.

**Comment by ymasmoudi** [comment]
@mpgermano we are deleting the certs after installation for now and we will figure out another solution

**Review by AyliD — APPROVED** [review]

File: xwf/gateway/deploy/roles/xwfm/tasks/main.yml [hunk 1]
```diff
@@ -18,16 +18,44 @@
...
   ignore_errors: true
 
-- name: set the hostname now
-  become: true
-  shell: hostname {{ 'xwfm.' + partner + '.' + env + '.' + server_id | default('1') }} 
+- name: display script output
+  debug:
+    msg: "{{ output }}"
 
 - name: delete ovs bridges incase they exist
```
Comment (mpgermano) [inline xwf/gateway/deploy/roles/xwfm/tasks/main.yml:64]: We need to remove the cert here as well

Comment (ymasmoudi) [reply xwf/gateway/deploy/roles/xwfm/tasks/main.yml:64]: @mpgermano added.

**Review by mpgermano — APPROVED** [review]

**Pull request merged**